firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/internal/metadata/MetaDataStoreTest.java

```diff
@@ -104,9 +104,10 @@ public void testWriteUserData_null() {
     assertNull(userData.getUserId());
   }
 
-  public void testWriteUserData_emptyString() {
+  public void testWriteUserData_emptyString() throws Exception {
```

Comment:
Oh test I have fixed in my pr, I think you can ignore the failed unit tests in metatDataStoreTest.

Reply:
Consider it ignored.